mappers/windows-virtual-exec/resource/windows-virtual-exec-instance.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: devices.kubeedge.io/v1alpha2
```

Comment:
It seems that this mapper does not use the v1beta1 version of the device API we proposed in version 1.15?https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/device-crd-v1beta1.md
And which version of kubeedge does this mapper run on?

Reply:
Currently we're running it on edgecore v1.14. And we are migrating our KE version to v1.18 however we have just upgraded our Linux nodes last week.

Indeed we will improve the mapper to suit the Things-oriented design introduced in v1beta1 api. Maybe we can update it asap.

Reply:
Currently we're running it on edgecore v1.14. And we are migrating our kubeedge version to v1.18 however we have just upgraded our Linux nodes last week.

Indeed we will improve the mapper to suit the Things-oriented design introduced in v1beta1 api. Maybe we can update it asap.